src/Tribe/Integrations/WPML/Meta.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * Translate post ids in Event meta data.
+ *
+ * @since TBD
+ */
+class Tribe__Events__Integrations__WPML__Meta {
+
+	/**
+	 * @since TBD
+	 *
+	 * @var Tribe__Events__Integrations__WPML__Meta
+	 */
+	protected static $instance;
+
+	/**
+	 * @since TBD
+	 *
+	 * @return Tribe__Events__Integrations__WPML__Meta
+	 */
+	public static function instance() {
+		if ( empty( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Translates post id in the Event meta data.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $value
+	 * @param int    $object_id
+	 * @param string $meta_key
+	 *
+	 * @return string The translated id for _EventOrganizerID & _EventVenueID.
+	 */
+	public function translate_post_id( $value, $object_id, $meta_key ) {
+
+		if ( isset( $_POST ) ) {
+			return;
```

Comment:
This needs to return `$value` otherwise it has the potential to break all `get_post_metadata` in a POST request.